### PR TITLE
Vulnerability Detector: NVD API 2.0

### PR DESCRIPTION
**Pull request by mateocervilla**
|Related issue|
|---|
|#6218|

## Description

After the following changes in the NVD: https://github.com/wazuh/wazuh/issues/16555, it is necessary to deprecate the tag `update_from_year` and to modify the offline update process for the NVD from Vulnerability Detector.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns.

### CODE REVIEW

**Review by MarcelKemp — APPROVED** [review]
LGTM.

File: source/user-manual/capabilities/vulnerability-detection/offline-update.rst [hunk 1]
```diff
@@ -361,50 +361,24 @@ Alternatively, they also can be loaded from a local path as follows:
...
-       <enabled>yes</enabled>
-       <path>/local_path/nvd-feed/nvd-feed[[:digit:]]\{4\}\.json\.gz$</path>
-       <update_interval>1h</update_interval>
+      <enabled>yes</enabled>
+      <url>http://local_repo/complete_nvd_feed.json.gz</url>
+      <update_interval>1h</update_interval>
    </provider>
 
-If you want to upload these files to a local server, they must follow the same numeric sequence that the script downloaded them in. To indicate the numerical range, you must indicate their position with the ``[-]`` tag helped by the ``start`` and ``end`` attributes. For example, you can use the configuration below if the files are from 2015 to 2020.
+To update locally, you have to set the path of the downloaded files with a POSIX regular expression as such:
```
Comment (s-ocando) [inline source/user-manual/capabilities/vulnerability-detection/offline-update.rst:374]:
```suggestion
To update locally, set the path of the downloaded files using a POSIX regular expression, for example:
```

**Review by s-ocando — COMMENTED** [review]
I have reviewed the PR, and everything looks good. I just have a small suggestion.

File: source/user-manual/capabilities/vulnerability-detection/offline-update.rst (hunk 1 above)
Comment (mateocervilla) [reply source/user-manual/capabilities/vulnerability-detection/offline-update.rst:374]: Should I also change the MSU section? I copy paste the description from there:

https://github.com/wazuh/wazuh-documentation/blob/74d70888e9305f077478b1037a4929f3905a43ad/source/user-manual/capabilities/vulnerability-detection/offline-update.rst?plain=1#L399

Comment (s-ocando) [reply source/user-manual/capabilities/vulnerability-detection/offline-update.rst]: It's not necessary to merge this PR, but it's a good opportunity to improve the text if you want to.

**Pull request merged**